config/runtime.exs

```diff
@@ -47,7 +47,9 @@ config :starknet_explorer, rpc_host: rpc_host
 
 config :starknet_explorer,
   rpc_host: rpc_host,
-  s3_bucket_name: "kraken-proofs"
+  s3_bucket_name: "kraken-proofs",
+  prover_storage: System.get_env("PROVER_STORAGE"),
+  proofs_root_dir: System.get_env("PROOFS_ROOT_DIR")
```

Comment:
This will silently be set as nil if the env variable is not found, do we want that behaviour? O do we want it to fail if elixir does not found said env variables?

Reply:
Added a default value to make it a little cleaner